packages/compiler/src/compiler/compile.ts

```diff
@@ -1,28 +1,28 @@
-import type { Node as LogicalExpression } from 'estree'
-
 import {
   CUSTOM_MESSAGE_ROLE_ATTR,
   CUSTOM_MESSAGE_TAG,
   REFERENCE_PROMPT_ATTR,
   REFERENCE_PROMPT_TAG,
-} from '../constants'
-import CompileError, { error } from '../error/error'
-import errors from '../error/errors'
-import parse from '../parser/index'
-import type {
-  BaseNode,
-  EachBlock,
-  ElementTag,
-  IfBlock,
-  TemplateNode,
-} from '../parser/interfaces'
+  TOOL_CALL_TAG,
+} from '$/constants'
```

Comment:
namespace alias with `$compiler/*` we need this to run the package from nextjs and api apps

Reply:
What do you mean

Reply:
in tsconfig.ts